### ISSUE TITLE
I got this error When i try to run the Iphone 8 plus simulator self	connectivity_plus.PathMonitorConnectivityProvider	0x00006000013f3ab0

### ISSUE THREAD

**Issue by vinodlee** [fluttercommunity/plus_plugins#2152]
### What is your question?

I got this error When i try to run the Iphone 8 plus simulator self	
connectivity_plus.PathMonitorConnectivityProvider	0x00006000013f3ab0

How can I fix this?

### Checklist before submitting a question

- [X] I Google'd a solution and I couldn't find it
- [X] I searched on StackOverflow for a solution and I couldn't find it
- [X] I read the README.md file of the plugin
- [X] I am using the latest version of the plugin
- [X] All dependencies are up to date with `flutter pub upgrade`
- [X] I did a `flutter clean`
- [X] I tried running the example project

**Comment by chenenyu** [comment]
Using Xcode15 ?

**Comment by vinodlee** [comment]
> Using Xcode15 ?

Yes

**Comment by vinodlee** [comment]
How can I fix this issue?

**Comment by VictorUvarov** [comment]
Xcode 15 made the minimum iOS version 12. I had 11 as the minimum. Updating my targets to 12.0 fixed this issue for me.

<img width="785" alt="Screenshot 2023-09-20 at 9 05 22 AM" src="https://github.com/fluttercommunity/plus_plugins/assets/35702719/bd77dd9f-2694-4dd2-9353-281106468889">


```ruby
platform :ios, '12.0'
```

**Comment by vinodlee** [comment]
> Xcode 15 made the minimum iOS version 12. I had 11 as the minimum. Updating my targets to 12.0 fixed this issue for me.
> 
> Targets: <img alt="Screenshot 2023-09-20 at 9 05 22 AM" width="785" src="https://user-images.githubusercontent.com/35702719/269348720-bd77dd9f-2694-4dd2-9353-281106468889.png">
> 
> Podfile:
> 
> ```ruby
> platform :ios, '12.0'
> ```

After updating the platform :ios, '12.0' I got the same error.

**Comment by chenenyu** [comment]
> > Xcode 15 made the minimum iOS version 12. I had 11 as the minimum. Updating my targets to 12.0 fixed this issue for me.
> > Targets: <img alt="Screenshot 2023-09-20 at 9 05 22 AM" width="785" src="https://user-images.githubusercontent.com/35702719/269348720-bd77dd9f-2694-4dd2-9353-281106468889.png">
> > Podfile:
> > ```ruby
> > platform :ios, '12.0'
> > ```
> 
> After updating the platform :ios, '12.0' I got the same error.

Update the minimum version to '12.0' both in xcode and Podfile

**Comment by vinodlee** [comment]
> > > Xcode 15 made the minimum iOS version 12. I had 11 as the minimum. Updating my targets to 12.0 fixed this issue for me.
> > > Targets: <img alt="Screenshot 2023-09-20 at 9 05 22 AM" width="785" src="https://user-images.githubusercontent.com/35702719/269348720-bd77dd9f-2694-4dd2-9353-281106468889.png">
> > > Podfile:
> > > ```ruby
> > > platform :ios, '12.0'
> > > ```
> > 
> > 
> > After updating the platform :ios, '12.0' I got the same error.
> 
> Update the minimum version to '12.0' both in xcode and Podfile

I have updated my Xcode version also but I got the same error.

<img width="960" alt="Screenshot 2023-09-21 at 3 20 08 PM" src="https://github.com/fluttercommunity/plus_plugins/assets/22536559/0be0870c-9406-458d-b7dc-54a61e5e5ffe">

**Comment by chenenyu** [comment]
Search `IPHONEOS_DEPLOYMENT_TARGET` in `project.pbxproj`

**Comment by florentin89** [comment]
@chenenyu your solution doesn't fix the problem for everyone. Maybe it was just for you. We all tried happy your solution but apparently the error persist for the rest of us. Will be nice if anyone could find a solution for all of us.

**Comment by chenenyu** [comment]
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    flutter_additional_ios_build_settings(target)

    target.build_configurations.each do |config|
      config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '12.0'
...
```

**Comment by vbuberen** [comment]
Missed this issue among other similar ones. 
There will be a release with fix #2169 which does the same thing - bumps min iOS to 12 for `network_info_plus` plugins.
